File: app/src/main/java/com/song/deviceinfo/ui/camera/CameraViewModel.java
package com.song.deviceinfo.ui.camera;

import android.content.Context;

import com.song.deviceinfo.info.CameraInfo;
import com.song.deviceinfo.ui.base.NormalViewModel;

import java.util.List;

import androidx.core.util.Pair;

/**
 * Created by chensongsong on 2020/9/30.
 */
public class CameraViewModel extends NormalViewModel {

    public List<Pair<String, String>> getCameraInfo(Context context) {
        return CameraInfo.getCameraInfo(context);
    }

}

File: app/src/main/java/com/song/deviceinfo/ui/camera/CameraViewModel.kt
package com.song.deviceinfo.ui.camera

import android.content.Context
import androidx.core.util.Pair
import com.song.deviceinfo.info.CameraInfo
import com.song.deviceinfo.ui.base.NormalViewModel

/**
 * Created by chensongsong on 2020/9/30.
 */
class CameraViewModel : NormalViewModel() {
    fun getCameraInfo(context: Context?): List<Pair<String, String>> {
        return CameraInfo.getCameraInfo(context!!)
    }
}
